mock-builder/src/location.rs

```diff
@@ -2,9 +2,27 @@ use frame_support::StorageHasher;
 
 use super::util::TypeSignature;
 
+/// Indicate how to perform the localtion hash
+/// See `FunctionLocation::hash()`
+#[derive(Clone, Copy, Debug)]
+pub enum TraitInfo {
+	/// Create has with trait info, panics if it has not.
```

Comment:
Typo and nit grammar
```suggestion
	/// Create hash with trait info, panics if it has none.
```